server.go

```diff
@@ -15,6 +17,16 @@ func main() {
 		s.Engine.Use(server.RequestLoggerHandler)
 	}
 
+	s.Engine.Use(cors.Middleware(cors.Config{
+		Origins:         "*",
+		Methods:         "GET, PUT, POST, DELETE",
+		RequestHeaders:  "Origin, Authorization, Content-Type",
+		ExposedHeaders:  "",
```

Comment:
Maybe add "ETag" and "Last-Modified". I don't think we create these either, but just something to keep in mind.